docs/tutorial/quick-start.md

```diff
@@ -55,45 +56,49 @@ The main script may look as follows:
 
 ```javascript fiddle='docs/fiddles/quick-start'
 const { app, BrowserWindow } = require('electron')
+const path = require('path')
 
 function createWindow () {
   const win = new BrowserWindow({
     width: 800,
     height: 600,
     webPreferences: {
-      nodeIntegration: true
+      preload: path.join(__dirname, 'preload.js')
     }
   })
 
   win.loadFile('index.html')
 }
 
-app.whenReady().then(createWindow)
+app.whenReady().then(() => {
+  createWindow()
+
+  app.on('activate', () => {
+    if (BrowserWindow.getAllWindows().length === 0) {
+      createWindow()
+    }
+  })
+)
```

Comment:
Hi, just found that this line missed a `}` when I followed the quick start guide to build the hello world app.
And it seems that only this PR is still missing the `}`.

Reply:
Thank you for pointing this out! I've sent #28253 to fix the issue in the `master` branch, and I'll also push updates for this PR